hooks/useGroupName.ts

```diff
@@ -1,22 +1,41 @@
 import type { ConversationTopic } from "@xmtp/react-native-sdk";
 import { currentAccount } from "../data/store/accountsStore";
+import { useGroupMembersQuery } from "../queries/useGroupMembersQuery";
 import { useGroupNameMutation } from "../queries/useGroupNameMutation";
 import { useGroupNameQuery } from "../queries/useGroupNameQuery";
+import { usePreferredNames } from "./usePreferredNames";
 
 export const useGroupName = (topic: ConversationTopic | undefined) => {
   const account = currentAccount();
-  const { data, isLoading, isError } = useGroupNameQuery({
+
+  const {
+    data: groupName,
+    isLoading: groupNameLoading,
+    isError,
+  } = useGroupNameQuery({
     account,
     topic: topic!,
   });
+
+  const { data: members, isLoading: membersLoading } = useGroupMembersQuery(
+    account,
+    topic!
```

Comment:
I think this should only be enabled if !groupName

Reply:
Yes you're right!